build-info-extractor-pip/src/main/java/org/jfrog/build/extractor/pip/extractor/PipInstall.java

```diff
@@ -0,0 +1,113 @@
+package org.jfrog.build.extractor.pip.extractor;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.jfrog.build.api.Build;
+import org.jfrog.build.api.util.Log;
+import org.jfrog.build.extractor.buildTool.BuildToolExtractor;
+import org.jfrog.build.extractor.buildTool.BuildToolUtils;
+import org.jfrog.build.extractor.clientConfiguration.ArtifactoryClientBuilderBase;
+import org.jfrog.build.extractor.clientConfiguration.ArtifactoryClientConfiguration;
+import org.jfrog.build.extractor.clientConfiguration.ArtifactoryDependenciesClientBuilder;
+import org.jfrog.build.extractor.clientConfiguration.client.ArtifactoryDependenciesClient;
+import org.jfrog.build.extractor.pip.PipDriver;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.jfrog.build.extractor.buildTool.BuildToolUtils.createArtifactoryClientConfiguration;
+
+/**
+ * Created by Bar Belity on 09/07/2020.
+ */
+public class PipInstall extends BuildToolExtractor {
+    private static final long serialVersionUID = 1L;
+    private static final String ARTIFACTORY_PIP_API_START = "/api/pypi/";
+    private static final String ARTIFACTORY_PIP_API_END = "/simple";
+
+    ArtifactoryClientBuilderBase clientBuilder;
+    PipDriver pipDriver;
+    Path workingDir;
+    String repo;
+    Log logger;
+    Path path;
+    List<String> installArgs;
+    String username;
+    String password;
+    String module;
```

Comment:
I think all the above class members can be private,

Reply:
Agreed.